code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2476,17 +2476,39 @@
 /datum/reagent/growthserum/on_mob_life(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
 	. = ..()
 	var/newsize = current_size
-	switch(volume)
-		if(0 to 19)
-			newsize = 1.25*RESIZE_DEFAULT_SIZE
-		if(20 to 49)
-			newsize = 1.5*RESIZE_DEFAULT_SIZE
-		if(50 to 99)
-			newsize = 2*RESIZE_DEFAULT_SIZE
-		if(100 to 199)
+	var/list/erp_area = list(//BUBBER ADDITION START - CAPPING GROWTH SERUM
+		/area/centcom/interlink/dorm_rooms,
+		/area/centcom/holding/cafe/dorms,
+		/area/misc/hilbertshotel,
+		/area/station/commons/dorms,
```

Comment:
You could probably move the `SIZE_WHITELISTED_AREAS` define from `modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/size_items.dm` into the defines folder and use it here instead for consistency